lib/mail/message.ex

```diff
@@ -316,4 +325,18 @@ defmodule Mail.Message do
 
     mimetype_fn.(extension)
   end
+
+  defp do_get_attachments(messages, attachments) do
```

Comment:
I don't use the `do_xyz` style. Please promote this to the public api function signature